src/evict/evict_page.c

```diff
@@ -171,6 +139,17 @@ __wt_evict(WT_SESSION_IMPL *session,
 		__wt_evict_list_clear_page(session, ref);
 	}
 
+	/*
+	 * Track how long forcible eviction took. Immediately increment the
+	 * forcible eviction statistics, we might do an in-memory split and
+	 * not an eviction, which skips the other statistics.
+	 */
+	time_start = 0;			/* [-Wconditional-uninitialized] */
+	if (LF_ISSET(WT_EVICT_CALL_URGENT)) {
```

Comment:
IIRC, I vaguely recall thinking that since it was likely to fail (especially in the new world of immediate evictions), it wasn't part of the cost of eviction. I don't feel strongly at all, feel free to simply include that work in the timing without further discussion.